### PR TITLE
[TASK] Update documentation

### PR DESCRIPTION
**Pull request by tstahn**
- Replace old cli_dispatch.phpsh usage
- Update list of available command line options (and add missing ones)
- Update command line usage examples

### CODE REVIEW

**Comment by tstahn** [comment]
Hi @tomasnorre, here is a first draft of the updated documentation.

I copied the setup from current `master` as you suggested before I replaced all `cli_dispatch.phpsh` occurences with TYPO3 9 compatible syntax and updated a few bits that seemed most obviously outdated to me.

What we're still missing though:
- updated screenshots
- explanation/examples on how to create and configure the scheduler tasks

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ The script to use is this:
 
 ::
 
-   [pathToYourTYPO3Installation]/typo3/cli_dispatch.phpsh crawler_im
+   [pathToYourTYPO3Installation]/typo3/sysext/core/bin/typo3 crawler:crawler:buildqueue
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: That's not the right command.

This should be the crawler:buildqueue and at best with the typo3cms binary. That's at least what I tested it with. 
so
```
[pathToYourTYPO3Installation-composer-bin-dir]/typo3cms crawler:buildqueue
```

would be the correct one.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst [hunk 2]
```diff
@@ -145,7 +144,7 @@ To do the same with the CLI script you run this:
 
 ::
 
-   [pathToYourTYPO3Installation]/typo3/cli_dispatch.phpsh crawler_im 3 -d 1 -conf tx_staticpub_publish
+   [pathToYourTYPO3Installation]/typo3/sysext/core/bin/typo3 crawler:crawler:buildqueue --startpage 3 --depth 1 --conf tx_staticpub_publish
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: Similar to above, binary and target should be changed.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst [hunk 3]
```diff
@@ -26,14 +26,17 @@ like this:
 
 ::
 
-   * * * * * [pathToYourTYPO3Installation]/typo3/cli_dispatch.phpsh crawler > /dev/null
+   * * * * * [pathToYourTYPO3Installation]/typo3/sysext/core/bin/typo3 crawler:crawler:buildqueue > /dev/null
```
Comment (tomasnorre) [inline Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst]: See above

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 1 above)
Comment (tstahn) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: `vendor/bin/typo3` (or `typo3/sysext/core/bin/typo3`) execute just fine, however I updated the paths according to your example.

As long as we stay consistent, I'm fine with either solution.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 2 above)
Comment (tstahn) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: Done.

File: Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst (hunk 3 above)
Comment (tstahn) [reply Documentation/ExecutingTheQueue/ExecutingQueueWithCron-job/Index.rst]: Done.

File: Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst (hunk 1 above)
Comment (tomasnorre) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: Is this path also updated or only the others ?

Comment (tstahn) [reply Documentation/ExecutingTheQueue/BuildingAndExecutingQueueRightAway(fromCli)/Index.rst]: All of them now use `[pathToYourTYPO3Installation-composer-bin-dir]/typo3cms crawler:buildqueue`.

**Pull request merged**